### PR TITLE
B/better loging

### PR DESCRIPTION
**Pull request by worldofjoni**
# Pull Request for initial Implementation

### CODE REVIEW

File: backend/src/layer/data/flickr_api/api_request.rs
Comment (Whatsuup) [inline backend/src/layer/data/flickr_api/api_request.rs:1]: These functions changed. Changes will be resolved during merge.

File: backend/src/layer/data/flickr_api/api_request.rs [hunk 1]
```diff
@@ -88,11 +88,10 @@ impl ApiRequest {
     }
```
Comment (Whatsuup) [inline backend/src/layer/data/flickr_api/api_request.rs:89]: Google won't give me such short and easy solutions...

**Review by Whatsuup — COMMENTED** [review]
I expect these graphql request traces cause too much log traffic. Should we still log this information in another mode?

**Comment by worldofjoni** [comment]
@Whatsuup 
Trace is already the lowes mode.
We now only trace every request once. I think this is ok, because if you dont want to get all these messages, you can just disable the trace level.

File: backend/src/layer/trigger/graphql/server.rs [hunk 2]
```diff
@@ -164,14 +167,42 @@ async fn graphql_handler(
...
     request: GraphQLRequest,
 ) -> GraphQLResponse {
-    let request = request.into_inner().data(
-        headers
-            .get("Authorization")
-            .and_then(|h| h.to_str().ok())
-            .unwrap_or("")
-            .to_string() as AuthHeader,
+    let auth_header = headers
+        .get("Authorization")
```
Comment (worldofjoni) [inline backend/src/layer/trigger/graphql/server.rs:171]: constant?

**Review by Whatsuup — APPROVED** [review]

**Pull request merged**